controllers/grid/users/reviewer/form/AdvancedSearchReviewerForm.php

```diff
@@ -155,6 +170,8 @@ public function fetch($request, $template = null, $display = false)
                     $submissionContext->getPath(),
                     'users/reviewers'
                 ),
+                'authorAffiliations' => $authorAffiliations,
+                'authors' => $authors,
```

Comment:
This is still data class representation of the SelectReviewerListPanel, which does not render authors.

Authors data are used within AdvancedSearchReviewerContainer, so you can pass them via selectReviewerListData directly.

Reply:
`        $templateMgr->assign('selectReviewerListData', [
            'authors' => $authors,
            'labels' => [
                'showAll' => __('showAll'),
                'showLess' => __('showLess'),
                'submissionAuthorList' => __('submission.author.list'),
            ],
            'components' => [
                'selectReviewer' => $selectReviewerListPanel->getConfig(),
            ]
        ]);`
        
Okay so I have done this now is that what you had in mind?